wire-gson-support/src/test/java/com/squareup/wire/protos/alltypes/AllTypes.java

```diff
@@ -1742,7 +1742,7 @@ public int hashCode() {
       return result;
     }
 
-    public static final class Builder extends com.squareup.wire.Message.Builder<NestedMessage, Builder> {
+    public static final class Builder extends com.squareup.wire.Message.Builder<NestedMessage, NestedMessage.Builder> {
```

Comment:
Nope, it's a small regression. I wanna fix it by improving the resolution algorithm.